src/ArcherContainer.js

```diff
@@ -85,6 +84,19 @@ function computeCoordinatesFromAnchorPosition(
   }
 }
 
+const possibleShapes: Array<ValidShapeTypes> = ['arrow', 'circle'];
+
+const getEndShapeFromStyle = (shapeObj: LineType) => {
+  if (!shapeObj.endShape) {
+    return possibleShapes[0];
+  }
+
+  return (
+    Object.keys(shapeObj.endShape).filter(key => possibleShapes.includes(key))[0] ||
```

Comment:
If `endShape` is present, this grabs the first `key` in the `endShape` object that is included in the `possibleShapes` array. If no key is found, we again fall back to `arrow`.